ledger/eval.go

```diff
@@ -102,11 +111,15 @@ func (cs *roundCowState) Get(addr basics.Address, withPendingRewards bool) (basi
 }
 
 func (cs *roundCowState) Put(record basics.BalanceRecord) error {
+	return cs.PutWithCreatables(record, nil, nil)
+}
+
+func (cs *roundCowState) PutWithCreatables(record basics.BalanceRecord, newCreatables []basics.CreatableLocator, deletedCreatables []basics.CreatableLocator) error {
```

Comment:
Out of curiosity, do we ever call `PutWithCreatables` with more than one new or deleted asset/app? Could we replace these slices with pointers? Or perhaps just directly pass in a `modifiedCreatable`?

Reply:
good call, I replaced the slices with pointers.